### PR TITLE
Reset accumulatedDamage #808

### PR DESCRIPTION
**Pull request by Hrvt2**
add variable damageCooldownTicks to Actor, in UpdateActorState reset Actor's accumulatedDamage if thier damageCooldownTicks is greater than GAME_TICKS_PER_SECOND (60).

### CODE REVIEW

File: src/cdogs/actors.c [hunk 1]
```diff
@@ -100,6 +100,7 @@
 #define GRIMACE_MELEE_TICKS 19
 #define DAMAGE_TEXT_DISTANCE_RESET_THRESHOLD (ACTOR_W / 2)
 #define FOOTPRINT_MAX 8
+#define GAME_TICKS_PER_SECOND 60 // Assuming 60 ticks per second for accumulatedDamage reset
```
Comment (cxong) [inline src/cdogs/actors.c:103]: the game is 70FPS
please use FPS_FRAMELIMIT

Comment (Hrvt2) [reply src/cdogs/actors.c]: Fixed as advised

**Comment by cxong** [comment]
looks good thanks!

**Comment by Hrvt2** [comment]
No problem, thank for your assistance.
________________________________
Od: congusbongus ***@***.***>
Odoslané: piatok 6. decembra 2024 4:24
Komu: cxong/cdogs-sdl ***@***.***>
Kópia: Tomáš Horvát ***@***.***>; Author ***@***.***>
Predmet: Re: [cxong/cdogs-sdl] Reset accumulatedDamage #808 (PR #867)


looks good thanks!

**Comment by Hrvt2** [comment]
No problem, thank you for assistance.

**Pull request merged**